app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java

```diff
@@ -0,0 +1,195 @@
+package io.neurolab.fragments;
+
+import android.content.Intent;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.WindowManager;
+
+import com.github.mikephil.charting.charts.LineChart;
+import com.github.mikephil.charting.components.Legend;
+import com.github.mikephil.charting.components.XAxis;
+import com.github.mikephil.charting.components.YAxis;
+import com.github.mikephil.charting.data.Entry;
+import com.github.mikephil.charting.data.LineData;
+import com.github.mikephil.charting.data.LineDataSet;
+import com.github.mikephil.charting.highlight.Highlight;
+import com.github.mikephil.charting.interfaces.datasets.ILineDataSet;
+import com.github.mikephil.charting.listener.OnChartValueSelectedListener;
+import com.github.mikephil.charting.utils.ColorTemplate;
+
+import io.neurolab.R;
+import io.neurolab.main.MainActivity;
+
+public class MemoryGraphFragment extends AppCompatActivity implements OnChartValueSelectedListener {
```

Comment:
What's an activity doing in a fragment directory?

Reply:
Ah! Actually initially I thought of making this a Fragment, but opinion changed to make an Activity due to some limitations with the former. Will change the directory. Thanks.

Reply:
Any update on the change?